README.md

```diff
@@ -10,10 +12,30 @@
 
 4. Are there unexpected regions with higher stars (rare patterns)? Something non-trivial that may be discovered are "hidden gems" such as food cart street or local spots
 
+5. Are more active yelpers prone to giving higher ratings or more thorough reviews?
+
+6. Do some yelpers give generally negative/positive reviews?
+
+7. Are there common words present in reviews with similar ratings?
+
+8. How consistent are ratings with common keywords?
+
+9. Do business ratings change over time? Can this change be attributed to attributes in the review data?
+
+10. What is the highest rated type of food (i.e. what restaurant is likely to please most?)
+
+11. What type of business receives the best/worst ratings?
+
+12. Does the quantity of photos indicate the rating of a business?
```

Comment:
To keep the scope narrow, I think we're sticking to text data  (at least to start with)

Reply:
Sounds good. Probably best to keep the proposal small in scope and increase if needed.